docs/source/installation/linux.rst

```diff
@@ -39,13 +39,61 @@ To install LaTeX:
    sudo apt install texlive texlive-latex-extra texlive-fonts-extra \
    texlive-latex-recommended texlive-science texlive-fonts-extra tipa
 
+If you don't have python3-pip installed, install it:
+
+.. code-block:: bash
+   
+   sudo apt install python3-pip
+  
 .. note:: These instructions are also valid for other Debian-based
           distributions or distributions that use the ``apt`` package manager.
 
 
+Fedora/CentOS/RHEL
+*************
+
+To install cairo:
+
+.. code-block:: bash
+
+  sudo dnf install cairo-devel
+
+To install ffmpeg:
+
+Add RPMfusion repository if it's not already added:
+
+.. code-block:: bash
+
+   sudo dnf -y install https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm
+
+Install ffmpeg from RPMfusion repository:
+
+.. code-block:: bash
+
+   sudo dnf install ffmpeg
+
+Install python development headers in order to successfully build pycairo wheel:
+
+.. code-block:: bash
+
+   sudo dnf install python3-devel
```

Comment:
Shouldn't this come before the "to install cairo" line?

Reply:
Honestly, there is no difference. The main reason for installing 'python3-devel' is because it fails installing pycairo while installing manim requirements, see below:
https://stackoverflow.com/questions/11094718/error-command-gcc-failed-with-exit-status-1-while-installing-eventlet
The point is to have both components installed while installing the manim requirements, is there something I'm missing?